sundials27.rb

```diff
@@ -0,0 +1,46 @@
+class Sundials27 < Formula
+  desc "Nonlinear and differential/algebraic equations solver"
+  homepage "https://computation.llnl.gov/casc/sundials/main.html"
+  url "https://computation.llnl.gov/projects/sundials/download/sundials-2.7.0.tar.gz"
+  sha256 "d39fcac7175d701398e4eb209f7e92a5b30a78358d4a0c0fcc23db23c11ba104"
+
+  conflicts_with "sundials", :because => "this is an older version"
+
+  option "with-openmp", "Enable OpenMP multithreading"
+
+  depends_on "cmake" => :build
+  depends_on "suite-sparse"
```

Comment:
Just to be sure, this is the `suite-sparse` in core. In `octave.rb`, you're requesting the one from this tap.

Reply:
Yeah, this is dangerous but seem to have worked on my machine. Possibly homebrew is fine if you one has any suite-sparse? Renaming all formula to "*-openblas" would fix this but has other drawbacks :(